addons/headless/functions/fnc_handleSpawn.sqf

```diff
@@ -4,26 +4,26 @@
  * Handles AI spawn and requests a rebalance if applicable.
  *
  * Arguments:
- * 0: Object <OBJECT>
+ * 0: Unit <OBJECT>
  *
  * Return Value:
  * None
  *
  * Example:
- * [object] call ace_headless_fnc_handleSpawn
+ * [cursorObject] call ace_headless_fnc_handleSpawn
  *
  * Public: No
  */
 
-params ["_object"];
-TRACE_1("Spawn",_object);
+params ["_unit"];
+TRACE_1("Spawn",_unit);
 
-// Exit if HC transferring disabled or object not a unit (including unit inside vehicle) or is player
-if (!(_object in allUnits) || {isPlayer _object}) exitWith {};
+// Exit if unit is player
+if (isPlayer _unit) exitWith {};
 
-// Exit and blacklist if of blacklist type
-if ({_object isKindOf _x} count GVAR(blacklistType) > 0) exitWith {
-    _object setVariable [QXGVAR(blacklist), true];
+// Exit and blacklist if unit is UAV
+if (unitIsUAV _unit) exitWith {
+    _unit setVariable [QXGVAR(blacklist), true];
```

Comment:
We could move this into `FUNC(transferGroups)` by checking `unitIsUAV _x` in the loop. there is no reason specific reason to set the blacklist variable here imo.

Reply:
Agreed.